src/main/scala/io/iohk/ethereum/blockchain/sync/regular/BlockImporterService.scala

```diff
@@ -0,0 +1,14 @@
+package io.iohk.ethereum.blockchain.sync.regular
+
+import cats.data.NonEmptyList
+import io.iohk.ethereum.blockchain.sync.regular.BlockImporter.{BlockImportType, ImportFn, ResolvingBranch}
+import io.iohk.ethereum.domain.Block
+
+//not used atm, a part of the future ExecutionSync
+class BlockImporterService() {
```

Comment:
I'm not super convinced of the value of adding empty files like this.

Reply:
I agree, this is not supposed to go to the codebase, it was here for the sake of our discussions, that were based on those pieces